fs/dafsclient/rmtclient.cpp

```diff
@@ -91,11 +91,15 @@ static unsigned dafsConnectFailRetryTimeMs = defaultDafsConnectFailRetrySeconds
 
 MODULE_INIT(INIT_PRIORITY_DAFSCLIENT)
 {
+#ifdef _CONTAINERIZED
+    //MORE: This function is called too soon to read them from the configuration file.
```

Comment:
there is no clear use-case at the moment for a dafilesrv client inside HPCC (only outside from a different client, e.g. Java/Spark).
So at the moment the whole of rmtclient.cpp/rmtfile.cpp - could be compiled out if we wanted I think.
But, we may want to go via dafilesrv client connection in future for some reason, e.g. routing through for security reasons